manager.go

```diff
@@ -23,13 +23,16 @@ import (
 	"path/filepath"
 	"sort"
 
+	"github.com/madlambda/spells/errutil"
 	"github.com/mineiros-io/terramate/git"
 	"github.com/mineiros-io/terramate/hcl"
 	"github.com/mineiros-io/terramate/project"
 	"github.com/mineiros-io/terramate/stack"
 	"github.com/rs/zerolog/log"
 )
 
+const ErrDirtyRepo errutil.Error = "the repository is not clean"
```

Comment:
wdyt?

```suggestion
const ErrDirtyRepo errutil.Error = "ERR_GIT_REPOSITORY_NOT_CLEAN"
```

Reply:
Do you mean you want the error messages / logs to be machine-oriented by default? If so, then we need to change a lot of stuff because in Go the error messages are human-friendly and we have been following this pattern since the beginning.

Reply:
Some examples of human-friendly error sentinels from Go stdlib/os: https://cs.opensource.google/go/go/+/refs/tags/go1.17.6:src/internal/oserror/errors.go;drc=refs%2Ftags%2Fgo1.17.6;l=13

We can do something different, just explaining the rationale behind error sentinels that consist of a simple human readable message (with no upper cases,etc). It is usually done that way so it can be composed with other errors, forming a longer phrase explaining the whole issue.

Some more info [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) (as mentioned, logging is a different topic, but here we are talking about error values/sentinels).

Reply:
gotcha.... ;) sorry for the noise... go.... ;)